src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/OpenSslX509CertificateReader.cs

```diff
@@ -165,6 +165,22 @@ internal static bool TryReadX509Der(byte[] rawData, out ICertificatePal certPal)
         }
 
         internal static bool TryReadX509Pem(SafeBioHandle bio, out ICertificatePal certPal)
```

Comment:
The naming used is a bit confusing, that ReadX509Pem ends up calilng to PEM_read_bio_X509_AUX.  I realize it was like that previously, but now that we have both varieties, seems like we should try to match more closely the underlying APIs, and have ReadX509Pem and ReadX509PemAux rather than ReadX509Pem and ReadX509PemNoAux?
cc: @bartonjs

Reply:
I don't have a strong opinion about this. I didn't change the name of ReadX509Pem because it is the default we should use.
I added the NoAux suffix to indicate the limiting.
@bartonjs any preference?

Reply:
I similarly don't have a strong opinion; having concluded what you've now written down as the reasoning.